galvani/res2sqlite.py

```diff
@@ -602,6 +614,11 @@ def convert_arbin_to_sqlite(input_file, output_file):
     print("Vacuuming database...")
     s3db.executescript("VACUUM; ANALYZE;")
 
+    if output_file == ":memory:":
+        return s3db
+
+    s3db.close()
```

Comment:
You could probably use a [contextlib.ExitStack](https://docs.python.org/3/library/contextlib.html#contextlib.ExitStack) to handle closing the connection properly if there is an error in the function, but not closing it when you want to return the connection. However, that can come later, this is good enough for now.

Reply:
I created #110 for this